net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/User.xml

```diff
@@ -63,6 +63,10 @@
                                 <default></default>
                                 <Required>N</Required>
                         </chillispot_bw_max_down>
+                        <sessionlimit_max_session_limit type="IntegerField">
+                                <default></default>
```

Comment:
Better to remove the line.

Reply:
Sorry, I dont get it, you mean remove default?

Reply:
Empty default can be omitted.